frontend/public/style/_common.scss

```diff
@@ -66,7 +66,6 @@
 }
 
 .co-m-pane__heading {
-  align-items: center;
```

Comment:
@alecmerdler had added this to fix the case where a logo is present.

![screen shot 2018-05-21 at 3 29 08 pm](https://user-images.githubusercontent.com/895728/40326007-c475965a-5d0b-11e8-8808-8a142dd92fb9.PNG)

Probably want to keep it or move it to .co-m-pane__heading--logo?

Reply:
@rhamilto In what section are the logos included?

Reply:
Applications > Cluster Service Versions